lib/pluginmanager/proxy_support.rb

```diff
@@ -67,6 +68,9 @@ def configure_proxy
   proxy = get_proxy("http_proxy")
   if valid_proxy?(proxy)
     proxy_settings = extract_proxy_values_from_uri(proxy)
+    if proxy_settings[:host].nil?
+      raise StandardError.new("Invalid proxy `#{proxy}`. I parsed it to: #{proxy_settings}")
```

Comment:
To reviewers: I wasn't sure which exception to use, I saw `ValidationError` being used in other parts of the code, but that seems to require being defined explicitly. Let me know what would work better.